src/main/java/uk/gov/hmcts/ccd/domain/service/getevents/ClassifiedGetEventsOperation.java

```diff
@@ -0,0 +1,58 @@
+package uk.gov.hmcts.ccd.domain.service.getevents;
+
+import com.google.common.collect.Lists;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.ccd.domain.model.definition.CaseDetails;
+import uk.gov.hmcts.ccd.domain.model.std.AuditEvent;
+import uk.gov.hmcts.ccd.domain.service.common.SecurityClassificationService;
+
+import java.util.List;
+import java.util.Optional;
+
+import static java.util.Collections.singletonList;
+
+@Service
+@Qualifier("classified")
+public class ClassifiedGetEventsOperation implements GetEventsOperation {
+
+    private final GetEventsOperation getEventsOperation;
+    private final SecurityClassificationService classificationService;
+
+    public ClassifiedGetEventsOperation(@Qualifier("default") GetEventsOperation getEventsOperation,
+                                        SecurityClassificationService classificationService) {
+
+        this.getEventsOperation = getEventsOperation;
+        this.classificationService = classificationService;
+    }
+
+    @Override
+    public List<AuditEvent> execute(CaseDetails caseDetails) {
```

Comment:
can we rename these 2 methods to getEvents and the one that returns a single event getEvent?

Reply:
done